### ISSUE TITLE
Need help with bidding option

### ISSUE THREAD

**Issue by sheribaloch** [dreamsoftin/facebook_audience_network#113]
Sir, I have implemented code it shows ads on test device but not working on other devices. I think it is we need to implement using bidding on new apps. Is there any method to implement facebook new ads strategy using your package?

**Comment by lostidol355** [comment]
You can use google mobile ads package to serve Facebook ads through bidding , you just need to ads mediation adapter in build.gardle file

**Comment by nilotpalkapri** [comment]
> You can use google mobile ads package to serve Facebook ads through bidding , you just need to ads mediation adapter in build.gardle file

Can you give some reference article to implement properly, I have tried but native ad never works in google mobile ads like this package. Isn't it possible to implement bidding in this package??

**Comment by sheribaloch** [comment]
> > You can use google mobile ads package to serve Facebook ads through bidding , you just need to ads mediation adapter in build.gardle file
> 
> Can you please give some reference article to implement properly, I have tried but native ad never works in google mobile ads like this package. Isn't it possible to implement bidding in this package??

I have implemented mediation using https://pub.dev/packages/google_mobile_ads and it is working. However I have implemented banner and Interstitial ads.

**Comment by nilotpalkapri** [comment]
> > > You can use google mobile ads package to serve Facebook ads through bidding , you just need to ads mediation adapter in build.gardle file
> > 
> > 
> > Can you please give some reference article to implement properly, I have tried but native ad never works in google mobile ads like this package. Isn't it possible to implement bidding in this package??
> 
> I have implemented mediation using https://pub.dev/packages/google_mobile_ads and it is working. However I have implemented banner and Interstitial ads. Now AdMob has put limitation on ad serving :/ mediation is still working but I don't know when they will terminate my account. I think we should look for alternative like MAX or MoPub but no support for flutter from any other platform till now

Thanks for the reply. I found an alternative of this package. https://pub.dev/packages/native_admob_flutter
Working fine for all type of ads.

**Comment by FaiJu** [comment]
@sheribaloch @nilotpalkapri Hey, can anyone of you please give me any reference article to implement bidding in flutter. I have implemented facebook ads in my app using Facebook audience network plugin, but as bidding is now necassary to show ads in app, what process do you guys recommend me to follow and any reference article for how to implemnet bidding of facebook ads with Google admob ?

**Comment by yatinj30** [comment]
> @sheribaloch @nilotpalkapri Hey, can anyone of you please give me any reference article to implement bidding in flutter. I have implemented facebook ads in my app using Facebook audience network plugin, but as bidding is now necassary to show ads in app, what process do you guys recommend me to follow and any reference article for how to implemnet bidding of facebook ads with Google admob ?

Have you got the solution?

**Comment by faizanhub** [comment]
Yes, I have removed the Facebook audience plugin and integrated ads with https://pub.dev/packages/google_mobile_ads package. Then have done bidding with Facebook ads through the interface of Google Admob

**Comment by Zia-Ur-Rahman-Ch** [comment]
@FaiJu hey thanks for sharing your experience.
I have a question is it possible to integrate Facebook ads using google mobile ads package that you have used
But without using Google admob.?
Is there any possibilities of doin this.

**Comment by faizanhub** [comment]
No, its not possible as per my knowledge
